docs/reference/concepts/deployments.md

```diff
@@ -1,81 +1,51 @@
+For more information, see the [comprehensive worker configuration documentation](https://docs.improbable.io/reference/latest/shared/project-layout/introduction#configuration-file).
 
-### Launch a cloud deployment
+> **Note:** The `build` section of the worker configuration files are **not** used by the GDK for Unity.
 
-1. Launch a cloud deployment
+## Local deployment
 
-   * In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
-   * This command defaults to deploying to clusters located in the US. If you’re in Europe, add the `--cluster_region=eu` flag for lower latency.
-   * It’s done when you see `Deployment launched successfully` printed in your terminal output. This might take a couple of minutes.
+To start a local deployment, in your Unity Editor, select **SpatialOS** > **Local Launch**.
 
-**About the `spatial cloud launch` command**   
-`spatial cloud launch` deploys a project to the cloud. Its full syntax is:
+This starts a local version of the SpatialOS Runtime on your development machine together with all server-workers specified in the launch configuration you used.
 
-```
-spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>
-```
+<%(Callout message="Its done when you see the following message in the console window which opens:<br/><br/>```SpatialOS ready. Access the Inspector at https://localhost:21000/inspector```")%>
 
-where:
+### Start a standalone client worker
 
-  * `<assembly name>` identifies the worker assemblies to use. The name needs to conform to the following regex: `[a-zA-Z0-9_.-]{5,64}`
-  * `<launch configuration>` is the configuration file for the deployment. This project includes `default_launch.json`, which is intended for use with local deployments, and `cloud_launch.json`, which is for cloud deployments.
-  * `<deployment name>` is a name of your choice, which is used to identify the deployment. The name needs to conform to the following regex: `[a-z0-9_]{2,32}`
-  * `<snapshot file>` is the snapshot of the world you want to start from. See this [documentation]({{urlRoot}}/reference/concepts/snapshots) for further information.
+The GDK provides tooling for starting client workers from the Unity Editor. 
 
-### Launch the game client
+First, ensure that you have built the worker by selecting **SpatialOS** > **Build for local** > **UnityClient**. This may take a few minutes depending on the size of your Unity project.
 
-  1. Open the SpatialOS [Console](https://console.improbable.io/projects). You’ll see the project and the deployment you just created.
-  1. In the SpatialOS Console, select the deployment’s name to open the overview page.
-  1. Select **Launch**.
+Then, in your Unity Editor,  select **SpatialOS** > **Launch standalone client**.
```

Comment:
2 spaces before `select`

Reply:
Since you performed the last action in the editor, there is no need to mention this step will have to be done in the editor